src/raylib/shapes/Circle.cpp

```diff
@@ -14,17 +16,16 @@ namespace raylib
         {
             _x = x;
             _y = y;
+            _z = z;
```

Comment:
For all positional characteristics: use one Vector3 instead of three separate variables.